lib/tasks/court_report_due_reminder.rake

```diff
@@ -0,0 +1,12 @@
+desc "Send an email to volunteers when their court report is due in 1 week, run by heroku scheduler."
+task court_report_due_reminder: :environment do
+  Volunteer.where(active: true).where.not(case_assignments:nil).find_each do |volunteer|
```

Comment:
You should be able to do an `.includes(:case_assignments)` here so that next line isn’t another query.

Reply:
Thanks @jmkoni !  I updated it and fixed the linting stuff :)